src/pages/resultsView/ResultsViewPageStoreUtils.ts

```diff
@@ -232,6 +316,26 @@ export function compileMutations<
     return mutations;
 }
 
+export function compileStructuralVariants<
```

Comment:
mergeStructuralVariants?  concatStructuralVariants?

Reply:
I was using the same format what was used for mutations `compileMutations`(line 297)